docs/developer/actors/test.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Testing the actor (Rust)"
+title: "Test (Rust Module)"
```

Comment:
Why "module"?

Reply:
Only because this guide was for the smithy-based actors. I don't know if there's an actual easy way to "test" the WASI HTTP stuff, so I might have to punt on this page altogether

Reply:
Ahh gotcha 👍